README.md

```diff
@@ -20,6 +20,14 @@ azure-iot-samples-csharp provides a set of easy-to-understand, continuously-test
 
   **Note:** The samples can be compiled using the NET Core SDK 2.1 SDK if the language version of projects using C# 8 features are changed to `preview`.
 
+## Preview features
+
+Samples samples showing how to use the various preview features of Microsoft Azure IoT .NET SDK are present [here](https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/preview).
+
+The following features are currently in preview:
```

Comment:
This is good to have but just wondering if we will remember to remove them when we add the features to GA, giving an impression they are still in preview.

Reply:
I agree that this definitely brings on maintenance overhead.
I like the idea of linking the samples in our release notes but how do we help customers who navigate to this repo directly? Eg, how do we make the device streaming sample discoverable to someone who just read the doc and decided to look for a sample?

Reply:
We could just call out the Device Streaming and write about it is what I feel. That is the only preview feature that has been in preview for so long and is not seeing any future. For the rest, it feels like customers will see the release notes, find out what is new and try it out. Eventually it will be in GA. It will be very hard to maintain this readme. 

We can use the same wordings used in the readme of the main repo.